### PR TITLE
Tpt3 77

### PR DESCRIPTION
**Pull request by zemlyanoi**
List referrals by staff id

### CODE REVIEW

File: legacy-data-access/src/main/java/gov/ca/cwds/data/legacy/cms/entity/CaseAssignment.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * @author CWDS TPT-3 Team
+ */
+@Entity
+@DiscriminatorValue(value = "C")
+public class CaseAssignment extends BaseAssignment {
+
+  private static final long serialVersionUID = -2902356846202575651L;
+
+  @NotFound(action = NotFoundAction.IGNORE)
```
Comment (dmytro-palczewski-osi) [inline legacy-data-access/src/main/java/gov/ca/cwds/data/legacy/cms/entity/CaseAssignment.java]: On Case side this association is mapped as @OneToMany so here must be @ManyToOne

File: legacy-data-access/src/main/java/gov/ca/cwds/data/legacy/cms/entity/Case.java [hunk 2]
```diff
@@ -178,6 +196,11 @@
   @Column(name = "TICKLE_T_B")
   private Boolean tickleIndVar;
 
+  @NotFound(action = NotFoundAction.IGNORE)
+  @OneToMany
```
Comment (dmytro-palczewski-osi) [inline legacy-data-access/src/main/java/gov/ca/cwds/data/legacy/cms/entity/Case.java]: add attribute  mappedBy="caseAssignment" to @OneToMany

File: legacy-data-access/src/main/java/gov/ca/cwds/data/legacy/cms/entity/ReferralAssignment.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+public class ReferralAssignment extends BaseAssignment {
+
+  private static final long serialVersionUID = 5323366175816010628L;
+
+  public static final String FIND_ASSIGNMENTS_BY_REFERRALS =
+      "ReferralAssignment.findAssignmentsByReferrals";
+  public static final String PARAM_REFERRAL_IDS = "referralIds";
+
+  @NotFound(action = NotFoundAction.IGNORE)
+  @OneToOne(fetch = FetchType.LAZY)
```
Comment (dmytro-palczewski-osi) [inline legacy-data-access/src/main/java/gov/ca/cwds/data/legacy/cms/entity/ReferralAssignment.java]: should Be changed to @ManyToOne

File: legacy-data-access/src/main/java/gov/ca/cwds/data/legacy/cms/entity/CaseAssignment.java (hunk 1 above)
Comment (zemlyanoi) [reply legacy-data-access/src/main/java/gov/ca/cwds/data/legacy/cms/entity/CaseAssignment.java]: done

**Pull request merged**